1970-01-01T00:33:41+00:00	docker.logs	{"log":"time=\"2021-12-01T11:24:45Z\" level=error msg=\"Port 30000 for service reactapp-service is already opened by another service\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:24:47.729297       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:24:49.569239 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:24:54.352839       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:24:54.353007       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/front-proxy-ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:24:54.684431       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:24:59.568339 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:24:59.637955       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:25:00.353354       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:25:02.826134       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/apiserver.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:25:02.826134       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:25:02.826179       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/front-proxy-ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:25:02.827946       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/front-proxy-client.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:25:02.841082       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:25:02.846621       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/front-proxy-ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:25:06.975081       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:25:06.976032       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:25:06.976188       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:25:06.976255       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:25:09.568887 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:25:11.132206       1 client.go:360] parsed scheme: \"passthrough\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:25:11.132384       1 passthrough.go:48] ccResolverWrapper: sending update to cc: {[{https://127.0.0.1:2379  <nil> 0 <nil>}] <nil> <nil>}\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:25:11.132422       1 clientconn.go:948] ClientConn switching balancer to \"pick_first\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:25:11.135163 I | embed: rejected connection from \"127.0.0.1:64448\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:25:11.136246       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:25:12.138155 I | embed: rejected connection from \"127.0.0.1:64456\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:25:12.138737       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:25:13.848087 I | embed: rejected connection from \"127.0.0.1:64478\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:25:13.849868       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:25:14.009595       1 horizontal.go:227] failed to query scale subresource for Deployment/default/dj-app-deployment: deployments/scale.apps \"dj-app-deployment\" not found\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:25:14.010992       1 event.go:291] \"Event occurred\" object=\"default/dj-app-hpa\" kind=\"HorizontalPodAutoscaler\" apiVersion=\"autoscaling/v2beta2\" type=\"Warning\" reason=\"FailedGetScale\" message=\"deployments/scale.apps \\\"dj-app-deployment\\\" not found\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:25:14.017106       1 horizontal.go:227] failed to query scale subresource for Deployment/default/fl-app-deployment: deployments/scale.apps \"fl-app-deployment\" not found\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:25:14.017229       1 event.go:291] \"Event occurred\" object=\"default/fl-app-hpa\" kind=\"HorizontalPodAutoscaler\" apiVersion=\"autoscaling/v2beta2\" type=\"Warning\" reason=\"FailedGetScale\" message=\"deployments/scale.apps \\\"fl-app-deployment\\\" not found\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"time=\"2021-12-01T11:25:15Z\" level=error msg=\"Port 30000 for service reactapp-service is already opened by another service\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:25:16.402525 I | embed: rejected connection from \"127.0.0.1:64498\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:25:16.402958       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:25:19.570535 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:25:20.416455 I | embed: rejected connection from \"127.0.0.1:64534\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:25:20.417143       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:25:26.808206 I | embed: rejected connection from \"127.0.0.1:64584\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:25:26.808855       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:25:29.646382 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:25:30.466571       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
